APIcall/data_collection.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import requests
+import argparse
+from datetime import date
+
+def api_311(start_date, end_date, skip, limit):
+    skip = 0
+    limit = 10000
+    data_year = []
+    while True:
+        url=f'https://dev-api.311-data.org/requests?start_date={start_date}&end_date={end_date}&skip={skip}&limit={limit}'
+        response = requests.get(url)
+        data = response.json()
+        if data == []:
+            break   
+        data_year.extend(data)
+        skip = skip + limit
+        data_final = pd.DataFrame(data_year) 
+        data_final.sort_values(by='createdDate', inplace = True, ignore_index = True)
+        data_final.to_csv('data_final.csv')
+    return data_final
+
+def main():
+    parser = argparse.ArgumentParser(description = 'API call to fetch 311 data from the pipeline')

```

Comment:
The description is a bit inaccurate, could be something like 'Gets 311 request data from the server'

Reply:
Yes, thanks Nich! Changed the description.